widget/src/apiService.js

```diff
@@ -1,7 +1,7 @@
 export default class APIService {
 
     static getEvents(context, callback) {
-        APIService.get(`${context.getApiUrl()}/current-events/verified?embed=Venue`, function(err, response) {
+        APIService.get(`${context.getApiUrl()}/events/current-verified?embed=Venue`, function(err, response) {
```

Comment:
good catch, you even updated the widget :-) I'm not sure if we use this for anything right now, but who knows maybe someday